doc/module-docs/cc_package_update_upgrade_install/example2.yaml

```diff
@@ -0,0 +1,7 @@
+#cloud-config
+package_update: true
+package_upgrade: true
+snap:
+  commands:
+    00: snap refresh --hold=forever
```

Comment:
We do include that comment in data.yaml for example2 which is rendered above the example in the modules documentation
`  - comment: "By default, `package_upgrade: true` will trigger upgrades on any installed package manager. To avoid calling `snap refresh` in images with snap installed, set snap refresh.hold to `forever` will prevent cloud-init's snap interaction during any boot"`. I'd updated this comment and the general desciprtion of the module to clarify that it interacts with any installed package managers which include apt/pkg/yum/zypper/snap